### PR TITLE
[Bug 1051539] English Article History page to local version raise 404

### PR DESCRIPTION
**Pull request by safwanrahman**
Similar to  #2651
The history page should show if any user requested with ``{{ host }}/{{ locale }}/kb/{{ english_article_slug }}/history``

I believe this should fix the issue! r?

### CODE REVIEW

**Comment by safwanrahman** [comment]
@mythmon r?

File: kitsune/wiki/tests/test_templates.py [hunk 1]
```diff
@@ -1039,6 +1039,25 @@ def test_history_category_appears(self):
...
+    def test_translation_history_with_english_slug(self):
+        """Request in en-US slug but translated locale should redirect to translation history"""
+        doc = DocumentFactory(locale=settings.WIKI_DEFAULT_LANGUAGE)
+        trans = DocumentFactory(parent=doc, locale='bn-BD', slug='bn_bd_trans_slug')
+        ApprovedRevisionFactory(document=trans)
+        # Get the page with the en-US slug
+        url = reverse('wiki.document_revisions', args=[doc.slug], locale=trans.locale)
+        response = self.client.get(url)
+        # Check redirection happens
+        eq_(302, response.status_code)
```
Comment (mythmon) [inline kitsune/wiki/tests/test_templates.py]: You should check that the location of the response matches what you'd expect. You can do this by checking the `Location` HTTP header with syntax like `response['Location']`.

**Comment by mythmon** [comment]
I'd like to see one more thing added to the tests, but besides that this looks good.

**Comment by safwanrahman** [comment]
@mythmon Travis passed! :green_apple: r?

**Pull request merged**